src/Entity/Version.php

```diff
@@ -635,6 +635,13 @@ public function getTags(): Collection
         return $this->tags;
     }
 
+    public function hasDevTag(): bool
+    {
+        return [] !== array_filter($this->getTags()->toArray(), static function (Tag $tag) {
```

Comment:
looping over the tags and stopping when we find a dev tag would be more efficient than filtering the whole list all the time.

Reply:
true